src/main/java/edu/harvard/iq/dataverse/util/json/JsonPrinter.java

```diff
@@ -788,4 +788,26 @@ public static JsonObjectBuilder getChecksumTypeAndValue(DataFile.ChecksumType ch
         }
     }
 
+    public static JsonObjectBuilder json(MapLayerMetadata mapLayerMetadata) {
+        Timestamp lastVerifiedTimestamp = mapLayerMetadata.getLastVerifiedTime();
+        String lastVerifiedString = null;
+        if (lastVerifiedTimestamp != null) {
+            lastVerifiedString = Util.getDateTimeFormat().format(lastVerifiedTimestamp);
+        }
+        return jsonObjectBuilder()
+                .add("id", mapLayerMetadata.getId())
+                .add("embedMapLink", mapLayerMetadata.getEmbedMapLink())
+                .add("joinDescription", mapLayerMetadata.getJoinDescription())
+                .add("layerLink", mapLayerMetadata.getLayerLink())
+                .add("layerName", mapLayerMetadata.getLayerName())
+                .add("mapImageLink", mapLayerMetadata.getMapImageLink())
+                .add("mapLayerLinks", Json.createReader(new StringReader(mapLayerMetadata.getMapLayerLinks())).readObject())
+                .add("worldmapUsername", mapLayerMetadata.getWorldmapUsername())
+                .add("isJoinLayer", mapLayerMetadata.isJoinLayer())
+                .add("dataFileId", mapLayerMetadata.getDataFile().getId())
```

Comment:
good idea but I put it in what I feel is a more useful place: https://github.com/IQSS/dataverse/blob/2ca06dca33e62a5a37c6ef1339e2a1d0b65c8229/src/main/java/edu/harvard/iq/dataverse/api/Geoconnect.java#L48